tests/engine/test_filter_macro_resolver.cpp

```diff
@@ -20,9 +20,27 @@ limitations under the License.
 using namespace std;
 using namespace libsinsp::filter::ast;
 
+static pos_info create_pos(uint32_t idx, uint32_t line, uint32_t col)
```

Comment:
This is good, but let's backport it to libs as a constructor/equality operator next!